wrappers/python/src/python/readdy/api/simulation.py

```diff
@@ -445,13 +445,14 @@ def current_topologies(self):
         """
         return self._simulation.current_topologies
 
-    def run(self, n_steps, timestep, show_system=True):
+    def run(self, n_steps, timestep, show_system=True, show_loop=True):
```

Comment:
What do you think about removing `show_loop` and `show_system` and instead adding a `summary` method printing information about the system and simulation

Reply:
agreed